runtime/src/lib.rs

```diff
@@ -1969,7 +1969,7 @@ impl oracle_proxy::Config for Runtime {
 }
 
 parameter_types! {
-    pub const GetBandRateStalePeriod: Moment = 60*5*1000; // 5 minutes
+    pub const GetBandRateStalePeriod: Moment = 60*10*1000; // 10 minutes
```

Comment:
should this also be replaced in mocks for consistency?

Reply:
Done